client.go

```diff
@@ -114,19 +132,7 @@ func New(address string, opts ...ClientOpt) (*Client, error) {
 		c.services = *copts.services
 	}
 	if address != "" {
-		backoffConfig := backoff.DefaultConfig
-		backoffConfig.MaxDelay = 3 * time.Second
-		connParams := grpc.ConnectParams{
-			Backoff: backoffConfig,
-		}
-		gopts := []grpc.DialOption{
-			grpc.WithBlock(),
-			grpc.WithTransportCredentials(insecure.NewCredentials()),
-			grpc.FailOnNonTempDialError(true),
-			grpc.WithConnectParams(connParams),
-			grpc.WithContextDialer(dialer.ContextDialer),
-			grpc.WithReturnConnectionError(),
-		}
+		gopts := DefaultDialOptions()
 		if len(copts.dialOptions) > 0 {
```

Comment:
Adding the passed dial options to the default list is another option, but it is a breaking change,